modules/ipi-verifying-nodes-after-installation.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// *
+
+:_module-type: PROCEDURE
+[id="ipi-verifying-nodes-after-installation_{context}"]
+= Verifying node state after installation
+
+[role="_abstract"]
+//Short introductory paragraph that provides an overview of the module. The text that immediately follows the abstract tag is used for search metadata.
+The {product-title} installation completes when the following installation health checks are successful:
+
+* The provisioning host can access the {product-title} web console.
+
+* All control plane nodes are ready.
+
+* All cluster Operators are available.
+
+[NOTE]
+====
+After the installation completes, the specific Cluster Operators responsible for the worker nodes continuously attempt to provision all worker nodes. It can take some time before all worker nodes report as `READY`. For installations on bare metal, wait a minimum of 60 minutes before troubleshooting a worker node. For installations on all other platforms, wait a minimum of 40 minutes before troubleshooting a worker node. A `DEGRADED` state for the Cluster Operators responsible for the worker nodes depends on the Operators' own resources and not on the state of the nodes.
+==== 
+
+After your installation completes, you can continue to monitor the condition of the nodes in your cluster using the following steps. 
+
+.Prerequisites
+//A bulleted list of conditions that must be satisfied before the user starts following this module.
+* The installation program resolves successfully in the terminal.
+
+.Procedure
+//Start each step with an active verb. Use an unnumbered bullet (*) if the procedure includes only one step.
+. Show the status of all worker nodes:
++
+[source,terminal]
+----
+oc get nodes
```

Comment:
This command needs a `$` prompt in front. Can you also fix for the below command as well?